pkg/event/event.go

```diff
@@ -51,7 +56,7 @@ type Event struct {
 	ID string `json:"id" example:"789be75d-7ac3-472e-bbbc-6d62878aad4a"`
 	// Type - The type of the occurrence which has happened.
 	// +required
-	Type string `json:"type" example:"event.synchronization-state-chang"`
+	Type string `json:"type" example:"com.redhat.cne.event"`
```

Comment:
leave it as event.synchronization-state-change , don't use redhat

Reply:
based on API spec defination

Reply:
done